cpg_workflows/stages/seqr_loader.py

```diff
@@ -225,6 +225,8 @@ def queue_jobs(self, dataset: Dataset, inputs: StageInput) -> StageOutput | None
         flag_name = str(outputs['done_flag'])
 
         job = get_batch().new_bash_job(f'Generate {index_name} from {mt_path}')
+        if not config_retrieve(['workflow', 'es_index', 'spot_instance'], default=True):
+            job = job.spot(is_spot=False)
```

Comment:
```suggestion
        job = job.spot(is_spot=config_retrieve(['workflow', 'es_index', 'spot_instance'], default=True))
```

This is a nit... current code is fine

Reply:
Fair to comment on, something was bugging me about the way it was 
`if not x, then y = false` - its just a bit confusing at a glance, IMO

I've changed it slightly, so it's now 
`if x is false, then y = false` 
And added an inline comment just so it's extra clear